profiles/kentik_snmp/cisco/cisco-firepower.yml

```diff
@@ -32,26 +28,105 @@ metrics:
         name: cpmCPUTotal1minRev
         tag: CPU
         poll_time_sec: 60
-  # Memory Utilization - polled at 60s to support anomaly detection
-  # Indicates the number of bytes from the memory pool that are currently in use by applications on the physical entity. This object is a 64-bit version of cempMemPoolUsed.
-  - MIB: CISCO-ENHANCED-MEMPOOL-MIB
-    symbol:
-      OID: 1.3.6.1.4.1.9.9.221.1.1.1.1.18.2.1
-      name: cempMemPoolHCUsed
-      tag: MemoryUsed
-      poll_time_sec: 60
-  # Memory Utilization - polled at 60s to support anomaly detection
-  # Indicates the number of bytes from the memory pool that are currently unused on the physical entity. This object is a 64-bit version of cempMemPoolFree.
-  - MIB: CISCO-ENHANCED-MEMPOOL-MIB
-    symbol:
-      OID: 1.3.6.1.4.1.9.9.221.1.1.1.1.20.2.1
-      name: cempMemPoolHCFree
-      tag: MemoryFree
-      poll_time_sec: 60
-  # Sessions Total - polled at 60s to support anomaly detection
-  # The number of currently active sessions.
-  - MIB: CISCO-REMOTE-ACCESS-MONITOR-MIB
-    symbol:
-      OID: 1.3.6.1.4.1.9.9.392.1.3.1.0
-      name: crasNumSessions
-      poll_time_sec: 60
+  - MIB: CISCO-FIREPOWER-SM-MIB
+    table:
+      OID: 1.3.6.1.4.1.9.9.826.1.71.20
+      name: cfprSmMonitorTable
+    symbols:
+      # Indicates the amount of unused memory on the security module.
+      - OID: 1.3.6.1.4.1.9.9.826.1.71.20.1.21
+        name: cfprSmMonitorMemFree
+        tag: MemoryFree
+        poll_time_sec: 60
+      # Indicates the amount of used memory on the security module.
+      - OID: 1.3.6.1.4.1.9.9.826.1.71.20.1.23
+        name: cfprSmMonitorMemUsed
+        tag: MemoryUsed
+        poll_time_sec: 60
+      # Indicates the amount of availabe disk space on the security module.
+      - OID: 1.3.6.1.4.1.9.9.826.1.71.20.1.17
+        name: cfprSmMonitorDataDiskAvailable 
+      # Indicates the total amount of disk space on the security module.
+      - OID: 1.3.6.1.4.1.9.9.826.1.71.20.1.18
+        name: cfprSmMonitorDataDiskTotal
+    metric_tags:
+      # The name of the security module.
+      - tag: module
```

Comment:
Since these modules come from different OID's we might want to tag the attribute with a more specific name, such as security_module, fan_module, and power_module

Reply:
Done